index.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
```

Comment:
do we need this top level index file? can we just drop this?

Reply:
From what I was able to tell when trying that, GitHub considers the pages a failed build if you do that. The file could be blank, I suppose, but I imagine it'll be used again in future tutorials.